glitch/analysis/security.py

```diff
@@ -174,3 +216,85 @@ def check_condition(self, c: ConditionStatement, file: str) -> list[Error]:
             return errors + [Error('sec_no_default_switch', c, file, repr(c))]
 
         return errors
+
+    def check_unitblock(self, u: UnitBlock) -> List[Error]:
+        errors = super().check_unitblock(u)
+
+        """
+        Missing integrity check changed to unit block since in Docker the integrity check is not an attribute of the
+        atomic unit but can be done on another atomic unit inside the same unit block.
+        """
```

Comment:
Usually these types of comments use (#) instead.
https://peps.python.org/pep-0008/#block-comments